### DISCUSSION TITLE
osc : No scheme for audioOut At 2 index, minimumVolume, and volumeAmplifier in Monterey Open Core 07.7

### DISCUSSION THREAD

**Discussion by ghost** [SkyrilHD/Dell-E7250-Hackintosh#51] [Q&A]
![Untitled](https://user-images.githubusercontent.com/89247060/150128594-6f017dfc-5b6e-4c31-82a3-845c274b0015.jpg)

**Comment by SkyrilHD** [comment]
The OpenCore version you use is not 0.7.7. Update your OpenCore!
These keys were replaced by AudioOutMask, MaximumAudibleGain and MaximumAssistGain.

**Reply by ghost** [reply]
thanks
